lab/lab2/js/part2-app-state.js

```diff
@@ -34,10 +34,19 @@
 ===================== */
 
 // We set this to HTTP to prevent 'CORS' issues
-var downloadData = $.ajax("http://");
-var parseData = function() {};
-var makeMarkers = function() {};
-var plotMarkers = function() {};
+var downloadData = $.ajax("https://raw.githubusercontent.com/CPLN692-MUSA611/datasets/master/json/philadelphia-crime-snippet.json");
+var parseData = function(data) {return JSON.parse(data)};
+var makeMarkers = function(parsed) {
+  var markers = [];
+  _.forEach(parsed, function(j){
+    markers.push(L.marker([j.Lat, j.Lng]))
+});
+  return markers;
+};
+var plotMarkers = function(markers) {
+  _.forEach(markers, function(k) {
+    k.addTo(map);
+  });
```

Comment:
![image](https://user-images.githubusercontent.com/8103418/53312189-145c2d80-3882-11e9-81a7-1083c907839c.png)

Looks like there is an error with a `{` somewhere. 

But otherwise, looks good.